core/run.py

```diff
@@ -91,6 +91,11 @@ def run():
 
     import logging
 
+    if getenv('ENVIRONMENT') == 'docker':
+        for skill in chain(MAX_WORKERS, ANNOTATORS, SKILL_SELECTORS, SKILLS):
```

Comment:
Shouldn't have `MAX_WORKERS` here, should have `RESPONSE_SELECTORS` and `POSTPROCESSORS`

Reply:
fixed